app/src/main/java/org/mozilla/focus/greenDAO/DownloadInfo.java

```diff
@@ -18,8 +18,9 @@
     private String Size;
     private String Date;
     private String FileName;
-    private String Uri;
+    private String MediaUri;
     private String MimeType;
+    private String FileUri;
 
     public DownloadInfo(long downloadId,String fileName){
```

Comment:
How many times will you be doing a database query when you load all data from DBService.getAllDownloadInfo?